sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -856,7 +864,7 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
         .getRepository(EducationProgramOffering)
         .save([requestedOffering, precedingOffering]);
 
-      // Save applications with new current assessment on approval.
+      // Save applications with new current assessment or status cancelled on approval.
```

Comment:
for my info, could you explain `or status cancelled`?

Reply:
When an offering change is approved by the ministry, 2 things can possibly take place. If the application status is completed, we create a new student assessment with the new approved offering and start the re-assessment workflow.
If the application is not complete, then the application status is set to cancelled and the student will be asked to re-submit the same application.

I know the comment is slightly confusing to explain what I mentioned above. I have modified the comment like this.

_**Save applications with new current assessment or set status as cancelled on approval.**_

Reply:
`Save applications with new current assessment or set application status as cancelled on approval.`

Reply:
Thanks for the explaination